app/Resources/assets/scss/main.scss

```diff
@@ -144,6 +144,24 @@ body#homepage {
 // Page: 'login'
 // --------------------------------------------------
 body#login {
+    #login-users-help p {
+        font-size: $font-size-base;
+        line-height: $line-height-base;
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+
+        .label {
+            margin-right: 5px;
+        }
+
+        .console {
```

Comment:
should this style be restricted to the login page ? We might want to display some help with commands in other pages too